src/modules/fancyzones/lib/util.h

```diff
@@ -96,6 +96,32 @@ namespace FancyZonesUtils
             SRCCOPY);
     }
 
+    inline COLORREF HexToRGB(const std::wstring& hex)
+    {
+        std::wstring color = hex;
+        if (hex.size() == 7)
+        {
+            color = hex.substr(1);
+        }
+        else if (hex.size() == 9)
+        {
+            color = hex.substr(3);
+        }
+
+        try
+        {
+            const auto tmp = std::stol(color, nullptr, 16);
+            const auto nR = (tmp & 0xFF0000) >> 16;
+            const auto nG = (tmp & 0xFF00) >> 8;
+            const auto nB = (tmp & 0xFF);
+            return RGB(nR, nG, nB);
+        }
+        catch (const std::exception&)
+        {
+            return RGB(255, 255, 255);
```

Comment:
let's give the ability to provide fallback color for the user and make this value a default for that arg

Reply:
Do you mean adding a setting for the fallback color so the users can choose it?

Reply:
no, just so we can call the function like this: `HexToRGB(hex, RGB(228, 322, 228));`. provide the fallback color basically, instead of hardcoding it inside the function. returning `std::optional` also works.

Reply:
I see, it's a nice to have, but in this case I prefer we set to white since this cover a case that is a bug and if it happens we want the users to notice something is wrong, and not to think that it simply didn't save the new value.

Reply:
yeah, that's why we could have a declaration like this:
`    inline COLORREF HexToRGB(const std::wstring& hex, const COLORREF fallbackColor = RGB(255, 255, 255))`

Reply:
OK, if we use RGB(255, 255, 255) as fallback that's fine with me.